utils/markdown.js

```diff
@@ -66,8 +65,9 @@
 /** @typedef {{
  * maxColumns?: number,
  * omitLastCR?: boolean,
- * flattenText?: boolean
- * renderCodeBlockTitlesInHeader?: boolean
+ * flattenText?: boolean,
+ * renderCodeBlockTitlesInHeader?: boolean,
+ * renderNoteAsText?: boolean,
```

Comment:
I'd recommend `notePrefix?: string, noteSuffix?: string`. This way, you can render code blocks or lists inside notes, even for typescript comments.